Frontend/src/GenericComponents/InputWithLabel/CollaboratorInput.jsx

```diff
@@ -63,8 +62,7 @@ function CollaboratorInput({ defaultValue, collaborators, setCollaborators, isEd
             setCollaborators(collaboratorsUpdate);
         } catch (error) {
             setErrorMessage('Email is invalid!');
-            toast.error(error);
-            console.error(error);
+            toast.error("An error occured while adding a collaborator, please retry");
         }
     }
 
```

Comment:
deleteCollaborator et handleRoleChange fait un call axios, tu ne vérifie pas le résultat.

Reply:
deleteCollaborator and handleRoleChanges ' call to axios are safely caught in case of an error